usecases/traverser/explorer.go

```diff
@@ -132,7 +132,7 @@ func (e *Explorer) getClassExploration(ctx context.Context,
 		res = withPath
 	}
 
-	return e.searchResultsToGetResponse(ctx, res, params.Explore.Certainty, searchVector)
+	return e.searchResultsToGetResponse(ctx, res, params.Explore.Certainty, searchVector, params)
```

Comment:
I think we can now remove the 3rd arg (`requiredCertainty`) as that's also contained in `params`.